src/main/java/com/su/gesipan/common/exception/handler/NotFoundHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.su.gesipan.common.exception.handler;
+
+import com.su.gesipan.common.exception.ErrorCode;
+import com.su.gesipan.post.PostNotFoundException;
+import com.su.gesipan.user.UserNotFoundException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+
+@Slf4j
+@ControllerAdvice
```

Comment:
```suggestion
@RestControllerAdvice
```
RestController이기 때문에 RestControllerAdvice로 잡아줘야 원하는 Error가 반환될것 같아요
지금 형태로 하면 500에러가 발생하네요!

Reply:
앗!!! 감사합니다!